Changes.md

```diff
@@ -55,6 +55,7 @@ Fixes
 - GraphEditor :
   - Improved handling of Backdrops when laying out nodes. Individual nodes are no longer pushed outside backdrops, and backdrops themselves are positioned as a group containing all their child nodes.
   - Improved positioning of pasted nodes, so that they are less likely to overlap with any node they are automatically connected to.
+- CyclesOptions : Fixed errors in section summaries.
```

Comment:
This is under the already-released 1.0.3.0 section, but needs to be under the not-yet-released 1.0.x.x section. I'll fix it up quickly while I merge, but something to keep an eye on next time...